ONSdigital/dp-frontend-renderer
# Feature/js enhance preview

### What

Updated the preview page template to allow for js enhanced creation of download files.

### How to review

Pull the following branches:

- Renderer - feature/js-enhance-preview
- Filter-controller - feature/js-enhance-preview
- sixteens - feature/js-enhance-preview

- Ensure that the files are generated without the need for a page refresh.
- Disable JS and check that a refresh page button is displayed if the downloads are still building.
- Stop a required service i.e. dp-dataset-exporter-xlsx - Check that an error is displayed after a couple of minutes.

### Who can review

Anyone.
